upt_macports/upt_macports.py

```diff
@@ -147,6 +151,14 @@ def _normalized_macports_folder(name):
     def jinja2_reqformat(self, req):
         return f'py${{python.version}}-{req.name.lower()}'
 
+    @property
+    def homepage(self):
+        homepage = self.upt_pkg.homepage
+        if homepage.startswith('http'):
+            return homepage
+        else:
+            return f'https://pypi.org/project/{self.upt_pkg.name}'
```

Comment:
Honestly, I should put this default value in upt-pypi. What do you think?

Reply:
sure, that wouldn't hurt! But one should make sure then that all situations where the metadata doesn't contain a valid URL are set to this value.